packages/node_modules/@ciscospark/widget-space/src/container.js

```diff
@@ -157,10 +157,6 @@ export class SpaceWidget extends Component {
     return nextProps !== this.props;
   }
 
-  componentWillUnmount() {
-    this.props.removeWidget();
```

Comment:
Is this handled elsewhere? We still need to remove the widget on unmount.

Reply:
So what the `.removeWidget` action did was clear the store. With the new base, that's not really an issue any more because of how we instantiate a new store. This was actually causing issues because if you try to compose this component with say a recents component inside the same app, it would wipe the complete store whenever the space component unmounted.

Reply:
Gotcha, I was thinking of the meet widget that shut down the camera access on remove.